mount/mount.cvmfs.cc

```diff
@@ -676,15 +680,17 @@ int main(int argc, char **argv) {
           return 32;
       }
     }
-  } while (stdout_open || stderr_open);
+    ended = (waitpid(pid_cvmfs, &status, 0) == pid_cvmfs);
```

Comment:
Shouldn't this use the `WNOHANG` option? Otherwise, there may be a deadlock here: the child process fills the stdout/stderr pipe with content and blocks. But the parent (== mount helper) does not read the data out of the pipe because it's waiting for the child to end.